### PR TITLE
New pm shortcode PM_BREADCRUMB

### PR DESCRIPTION
**Pull request by rica-carv**
Now pm breadcrumb is hardcoded and forced on every pm templates.

With this new pm shortcode {PM_BREADCRUMB}, it allows full customization of pm_template, with or without breadcrumb...

### CODE REVIEW

**Comment by CaMer0n** [comment]
Thanks. I'll come back to this after 2.1.6 is released.

File: e107_plugins/pm/pm.php [hunk 1]
```diff
@@ -882,7 +892,10 @@ function pm_user_lookup()
 	switch($action)
 	{
 		case 'send' :
-			$ns->tablerender(LAN_PM, $pm->breadcrumb(LAN_PLUGIN_PM_NEW) . $mes->render() . $pm->show_send($pm_proc_id));
+//--			$ns->tablerender(LAN_PM, $pm->breadcrumb(LAN_PLUGIN_PM_NEW) . $mes->render() . $pm->show_send($pm_proc_id));
+ 			$sc->bc_type = LAN_PM_55;
+   		$sc->bc_other = NULL;
+			$ns->tablerender(LAN_PM, $mes->render() . $pm->show_send($pm_proc_id));
 			break;
```
Comment (CaMer0n) [inline e107_plugins/pm/pm.php]: Shouldn't $sc->bc_type = LAN_PLUGIN_PM_NEW here?

Comment (CaMer0n) [reply e107_plugins/pm/pm.php]: @rica-carv I'd still like to get this merged. What are your thoughts on the comment above?

Comment (Moc) [reply e107_plugins/pm/pm.php]: @rica-carv any thoughts on Cameron's comment?

**Comment by rica-carv** [comment]
Can i have a feedback with this was closed without merging? Thanks.

File: e107_plugins/pm/pm.php (hunk 1 above)
Comment (rica-carv) [reply e107_plugins/pm/pm.php:899]: @CaMer0n @Moc Sorry, i only see now your questions... @CaMer0n You're right, it should be PM_NEW...

**Pull request closed without merging**